webui-src/styles/app.css

```diff
@@ -0,0 +1,7299 @@
%20relative;%5Cn%20%20top:%20100px;%5Cn%20%20max-width:%20400px;%5Cn%20%20max-height:%20500px;%5Cn%20%20border-radius:%205px;%5Cn%5Cn%20%20display:%20flex;%5Cn%20%20align-items:%20flex-start;%5Cn%20%20flex-direction:%20column;%5Cn%20%20align-items:%20center;%5Cn%7D%5Cn%5Cn.extra%20%3E%20label,%5Cn.extra%20%3E%20br,%5Cn.extra%20%3E%20input%20%7B%5Cn%20%20margin-bottom:%200;%5Cn%7D%5Cn%5Cn.login-container%20*%20%7B%5Cn%20%20margin-bottom:%2015px;%5Cn%7D%5Cn.login-container%20%3E%20img%20%7B%5Cn%20%20margin-top:%2015px;%5Cn%20%20margin-bottom:%2030px;%5Cn%7D%5Cn%5Cn.login-container%20extra%20%7B%5Cn%20%20margin:%200;%5Cn%7D%5Cn.login-container%20%3E%20a%20%7B%5Cn%20%20text-decoration:%20underline;%5Cn%20%20cursor:%20pointer;%5Cn%7D%5Cn%22,%22.certificate%20%7B%5Cn%20%20margin:%20auto;%5Cn%20%20display:%20flex;%5Cn%20%20align-items:%20flex-start;%5Cn%20%20flex-direction:%20column;%5Cn%20%20align-items:%20center;%5Cn%20%20margin-top:%2020px;%5Cn%7D%5Cn.textArea%20%7B%5Cn%20%20box-sizing:%20none;%5Cn%20%20background:%20transparent;%5Cn%20%20font-size:%201.1rem;%5Cn%20%20border-radius:%205px;%5Cn%20%20border:%20none;%5Cn%20%20resize:%20none;%5Cn%7D%5Cn.logo%20%7B%5Cn%20%20display:%20flex;%5Cn%20%20max-width:%20500px;%5Cn%20%20justify-content:%20center;%5Cn%20%20align-items:%20center;%5Cn%7D%5Cn.retroshareText%20%3E%20h3%20%7B%5Cn%20%20margin-right:%203px;%5Cn%7D%5Cn.webhelp%20%7B%5Cn%20%20background:%20whitesmoke;%5Cn%20%20font-size:%200.8rem;%5Cn%20%20border-radius:%205px;%5Cn%20%20display:%20flex;%5Cn%20%20justify-content:%20center;%5Cn%20%20align-items:%20center;%5Cn%20%20gap:%200.5rem;%5Cn%20%20color:%20black;%5Cn%20%20border-radius:%205px;%5Cn%20%20border:%201px%20solid%20black;%5Cn%20%20padding:%2010px%2014px;%5Cn%20%20cursor:%20pointer;%5Cn%7D%5Cn.webhelp%20%3E%20i%20%7B%5Cn%20%20font-size:%201.2rem;%5Cn%7D%5Cn.webhelp%20%3E%20p%20%7B%5Cn%20%20font-weight:%20600;%5Cn%7D%5Cn.retroshareID%20%7B%5Cn%20%20display:%20flex;%5Cn%20%20justify-self:%20start;%5Cn%20%20align-items:%20center;%5Cn%20%20font-size:%201.2rem;%5Cn%20%20font-family:%20monospace;%5Cn%7D%5Cn%22,%22.friend%20%7B%5Cn%20%20color:%20%23444;%5Cn%20%20font-size:%201.2em;%5Cn%20%20margin:%201em;%5Cn%20%20padding:%2020px;%5Cn%20%20border:%201px%20solid%20%23aaa;%5Cn%20%20border-radius:%2020px;%5Cn%7D%5Cn.friend%20i%20%7B%5Cn%20%20float:%20left;%5Cn%20%20padding:%200%2010px;%5Cn%20%20cursor:%20pointer;%5Cn%7D%5Cn.friend%20h4%20%7B%5Cn%20%20margin-bottom:%205px;%5Cn%7D%5Cn.friend%20p%20%7B%5Cn%20%20margin:%200;%5Cn%7D%5Cn.friend%20button%20%7B%5Cn%20%20font-size:%200.9em;%5Cn%7D%5Cn.friend.hidden%20%7B%5Cn%20%20display:%20none;%5Cn%7D%5Cn.friend%20.brief-info.online%20%7B%5Cn%20%20color:%20green;%5Cn%7D%5Cn%5Cn.friend%20.location%20%7B%5Cn%20%20margin:%205px;%5Cn%20%20border-top:%201px%20solid%20%23bbb;%5Cn%20%20display:%20grid;%5Cn%20%20grid-template-columns:%20auto%20auto;%5Cn%20%20justify-content:%20start;%5Cn%7D%5Cn.friend%20.brief-info%20%7B%5Cn%20%20display:%20flex;%5Cn%20%20align-items
```

Comment:
The build script command you added should create a compressed file, without sourcemaps, but this one doesn't seem to be it

`sass --no-source-map --style=compressed ./app/scss/main.scss ./styles/app.css`

Reply:
oops!, I actually made the script later so, forgot to update the app.css 😅

Reply:
Done Now : )

Reply:
Thanks!